Disable password change notification emails (#62)
alleyinteractive/wp-alleyvate: README.md

This feature removes clutter from the dashboard.

### `disable_password_change_notification`

> This feature disables the sending of password change notification emails.

Reviewer: Suggestion: make it obvious this is disabling the notifications to the admin.

Author: I think this as obvious as I can make it

Reviewer 2: I agree with Renato here, this could be confused with the emails that are sent to users when they _request_ a password change, vs. the notification to the admin when a user resets their password. Adding "to site admins" to the end of this sentence accomplishes that goal, imo
```suggestion
This feature disables sending password change notification emails to site admins.
```
